core/payment-driver/base/src/driver.rs

```diff
@@ -20,68 +20,93 @@ pub use ya_core_model::payment::local::Network;
 
 #[async_trait(?Send)]
 pub trait PaymentDriver {
+    /// Called by the Identity service to notify the driver that specified
+    /// account is _locked_ / _unlocked_. Identity service holds
+    /// accounts private keys and signs transactions.
     async fn account_event(
         &self,
         _db: DbExecutor,
         _caller: String,
         msg: IdentityEvent,
     ) -> Result<(), IdentityError>;
 
+    /// Gets the balance of the account.
     async fn get_account_balance(
         &self,
         db: DbExecutor,
         caller: String,
         msg: GetAccountBalance,
     ) -> Result<BigDecimal, GenericError>;
 
+    /// Deposits the funds into the driver's supported network. Called by CLI.
     async fn enter(
         &self,
         db: DbExecutor,
         caller: String,
         msg: Enter,
     ) -> Result<String, GenericError>;
 
+    /// Exits the funds outside the driver's supported network (most likely L1).
+    /// Called by CLI.
     async fn exit(&self, db: DbExecutor, caller: String, msg: Exit)
         -> Result<String, GenericError>;
 
     // used by bus to bind service
     fn get_name(&self) -> String;
     fn get_default_network(&self) -> String;
     fn get_networks(&self) -> HashMap<String, Network>;
+
+    /// Tells whether account initialization is needed for receiving payments.
     fn recv_init_required(&self) -> bool;
 
+    #[deprecated(
```

Comment:
nice idea! but you also need to patch the build so it allows for these deprecated warnings

Reply:
Ah in this case I think I will do with just a comment. Making the warning and then silencing it seems like solving unreal problems ;)